bin/setenv-docker-customize.sh

```diff
@@ -175,6 +176,11 @@ EXO_ES_URL="${EXO_ES_SCHEME}://${EXO_ES_HOST}:${EXO_ES_PORT}"
 
 [ -z "${EXO_ADDONS_CONFLICT_MODE}" ] && EXO_ADDONS_CONFLICT_MODE=""
 
+[ -z "${EXO_JCR_FS_STORAGE_ENABLED}" ] && EXO_JCR_FS_STORAGE_ENABLED=""
```

Comment:
So, if unset like here, the storage in standalone will be in FS?

Reply:
It will be up to eXo Server to define default values in fact.
If in docker env variables, not set, the product should manage the default values (Behavior defined by product)
Else, you can embed them in `docker-compose.yml` to be sure that it will remain the same even if the product changes its default values.